package.json

```diff
@@ -2,7 +2,7 @@
   "name": "detect-adblock",
   "version": "0.0.1",
   "description": "Implements detecting of adblockers in browser",
-  "main": "lib/index.js",
+  "main": "src/index.js",
```

Comment:
@selfagency when we are publishing package to npm, we are building it to `lib` directory which is ignored in git. So when you install pacakge from NPM, you will have lib directory in it. 

We don't add lib to repository, because we don't want to have problems with merge conflicts.